static/src/javascripts/projects/common/modules/audio/AudioPlayer.js

```diff
@@ -435,35 +438,22 @@ export default class AudioPlayer extends Component<Props, State> {
             const boxW = document.querySelector('.fake-wave').offsetWidth;
             const svg = document.querySelector('.fake-wave svg');
             // $FlowFixMe
-            const pt = svg.createSVGPoint();
-
-            pt.x = e.clientX;
-            pt.y = e.clientY;
-            const adjustedPosition = pt.matrixTransform(
-                // $FlowFixMe
-                svg.getScreenCTM().inverse()
-            );
-            const clickedPosition = (adjustedPosition.x / boxW) * 98; // sorry this just seemed to work
-            this.seek(clickedPosition);
-        }
-    };
+            const leftOffset = svg.getBoundingClientRect().left;
```

Comment:
Great detective work 🕵️‍♀️

Reply:
haha, I will leave the coordinate system alone unless I really have to